### PR TITLE
stone-phaser: init at 0.1.2

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/applications/audio/stone-phaser/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [
+    xorg.libX11 cairo lv2 libjack2 mesa
+  ];
+
+  patchPhase = ''
+    patch -d dpf -p 1 -i "`pwd`/resources/patch/DPF-bypass.patch"
```
Comment (vyp) [inline pkgs/applications/audio/stone-phaser/default.nix]:
```suggestion
    patch -d dpf -p 1 -i "resources/patch/DPF-bypass.patch"
```

Is `\`pwd\`` needed here? The unpack phase should have copied the src to the current directory I think.

Comment (magnetophon) [reply pkgs/applications/audio/stone-phaser/default.nix]: Seems like it is, without I get: ``patch: **** Can't open patch file resources/patch/DPF-bypass.patch : No such file or directory``.

Comment (vyp) [reply pkgs/applications/audio/stone-phaser/default.nix]: Oh interesting, my bad :+1:

**Comment by magnetophon** [comment]
@marsam Wanna take a look here as well?

File: pkgs/applications/audio/stone-phaser/default.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    sha256 = "180b32z8h9zi8p0q55r1dzxfckamnngm52zjypjjvvy7qdj3mfcd";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [
+    xorg.libX11 cairo lv2 libjack2 mesa
+  ];
+
+  patchPhase = ''
```
Comment (marsam) [inline pkgs/applications/audio/stone-phaser/default.nix]:
```suggestion
  postPatch = ''
```

File: pkgs/applications/audio/stone-phaser/default.nix (hunk 1 above)
Comment (marsam) [inline pkgs/applications/audio/stone-phaser/default.nix]:
```suggestion
    patch -d dpf -p 1 -i "$src/resources/patch/DPF-bypass.patch"
```

**Review by marsam — COMMENTED** [review]
I left a pair of comments. Besides that, LGTM

File: pkgs/applications/audio/stone-phaser/default.nix (hunk 1 above)
Comment (magnetophon) [reply pkgs/applications/audio/stone-phaser/default.nix]: Thanks for the feedback, all done!

**Review by marsam — APPROVED** [review]

**Pull request merged**